contentcuration/contentcuration/static/js/edit_channel/file_upload/views.js

```diff
@@ -181,12 +181,12 @@ var FileUploadList = FileBaseList.extend({
         this.acceptedFiles = this.get_accepted_files();
         this.onsave = options.onsave;
         this.onnew = options.onnew;
-        this.upload_in_progress = false;
+        this.uploads_in_progress = 0;
         this.render();
     },
     handle_if_empty:function(){
         this.$(this.default_item).css("display", (this.views.length > 0) ? "none" : "block");
-        this.disable_next(this.upload_in_progress);
+        this.disable_next(this.uploads_in_progress > 0);
```

Comment:
Possible solution to https://trello.com/c/3jugIhvx/195-upload-file-window-stuck-at-upload-in-progress-although-file-shows-as-uploaded-successfully